WordPress/Classes/Extensions/SwiftUI+Backport.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+
+struct Backport<T: View> {
+    let view: T
+}
+
+extension View {
+    var backport: Backport<Self> { Backport(view: self) }
+}
+
+extension Backport {
+    @ViewBuilder
+    func refreshable(action: @Sendable @escaping () async -> Void) -> some View {
+        if #available(iOS 15, *) {
+            view.refreshable(action: action)
```

Comment:
I will remove it once iOS 14 support is dropped – [already accepted](https://jetpackmobile.wordpress.com/2023/05/30/should-we-drop-ios-14-support/) 🎉.